packages/create-vite/template-vue-ts/package.json

```diff
@@ -11,7 +11,7 @@
   },
   "devDependencies": {
     "@vitejs/plugin-vue": "^1.6.0",
-    "@vue/compiler-sfc": "^3.0.5",
+    "@vue/compiler-sfc": "3.2.6",
```

Comment:
why remove the leading `^`?

p.s. you can prefix this PR with `deps:` to make it pass the semantic PR bot check

Reply:
OK, has solved, thanks!